buildout.cfg

```diff
@@ -13,6 +13,10 @@ recipe = zc.recipe.egg
 eggs =
     Sphinx
     docutils
+    ZODB
+    j1m.sphinxautointerface
```

Comment:
idk, I wasn't aware of that.  Looking at the docs, I don't see a way to control which members are included.  I fixed that in sphinxcontrib-zopeext, of which j1m.sphinxautointerface is a fork. See: https://bitbucket.org/birkenfeld/sphinx-contrib/pull-requests/126/fixes-based-on-using-the-extension-with/diff. I viewed this as a bug in the context of sphinxcontrib-zopeext, as that is based on autoclass, which allows member selection.

I want member selection because many of the ZODB interfaces address multiple audiences, and I want the ZODB docs to focus on ZODB users (people who develop with it, not people who, for example, create strorages.)